src/components/calcite-tile-select/calcite-tile-select.stories.ts

```diff
@@ -24,8 +24,8 @@ export const Light = (): string => html`
     heading="${text("heading", "Tile heading lorem ipsum")}"
     ${boolean("hidden", false)}
     icon="${select("icon", iconNames, iconNames[296])}"
-    name="${text("name", "tile-select-demo")}"
-    show-input="${select("show-input", ["left", "right", "none"], "left")}"
+    ${boolean("show-input", false)}
+    show-input="${select("input-alignment", ["start", "end"], "start")}"
```

Comment:
Attr needs to be updated, this is setting the alignment on `show-input`. There are other instances of this.

Reply:
Whoops, good catch.